external/iotjs/src/platform/tizenrt/iotjs_main-tizenrt.c

```diff
@@ -116,9 +116,11 @@
  pthread_addr_t iotjs_thread(void *thread_arg) {
    struct iotjs_thread_arg *arg = thread_arg;
 
-   iotjs_entry(arg->argc, arg->argv);
+   int ret = iotjs_entry(arg->argc, arg->argv);
    tuv_cleanup();
-
+#ifdef CONFIG_DEBUG
```

Comment:
1. We need to make IoT.js 's debug
2. This information is not critical so that we had better use CONFIG_DEBUG_VERBOSE.